pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/admin/UserAdminService.java

```diff
@@ -741,7 +738,7 @@ protected static Map<Integer, Set<String>> getRolesForStudyUsers(int groupId) {
         });
     }
 
-    protected static int getRoleId(String role) {
```

Comment:
I'm not comfortable with this approach (so this applies to many of the changes in this file). IMHO the right way to do this is to not expose the inner workings of this service -- and thus greatly expand its contract -- but to create a test helper class to provide what tests need (since this is not what the service needs to provide per se).
Does that make sense? Happy to discuss.